br/pkg/task/backup_ebs.go

```diff
@@ -253,6 +273,123 @@ func RunBackupEBS(c context.Context, g glue.Glue, cmdName string, cfg *BackupEBS
 	return nil
 }
 
+func waitAllScheduleStoppedAndNoRegionHole(ctx context.Context, cfg Config, mgr *conn.Mgr) error {
+	allStores, err := conn.GetAllTiKVStoresWithRetry(ctx, mgr.GetPDClient(), util.SkipTiFlash)
+	if err != nil {
+		return errors.Trace(err)
+	}
+	// we wait for 15*40 = 600s = 10m at least
+	for i := 0; i < 40; i++ {
+		if ctx.Err() != nil {
+			return ctx.Err()
+		}
+		if i > 0 {
+			time.Sleep(waitAllScheduleStoppedInterval)
+		}
+		allRegions, err2 := waitUntilAllScheduleStopped(ctx, cfg, allStores, mgr)
+		if err2 != nil {
+			if causeErr := errors.Cause(err2); causeErr == hasPendingAdminErr {
+				log.Info("schedule ongoing on tikv, will retry later", zap.Error(err2))
+			} else {
+				log.Warn("failed to wait schedule, will retry later", zap.Error(err2))
```

Comment:
backup never get success due to follower error.
I0911 11:06:18.799087       9 backup.go:267] [2022/09/11 11:06:18.798 +00:00] [WARN] [backup_ebs.go:329] ["failed to wait schedule, will retry later"] [error="rpc error: code = Unimplemented desc = "] [errorVerbose="rpc error: code = Unimplemented desc = \ngithub.com/pingcap/errors.AddStack\n\t/home/runner/go/pkg/mod/github.com/pingcap/errors@v0.11.5-0.20211224045212-9687c2b0f87c/errors.go:174\ngithub.com/pingcap/errors.Trace\n\t/home/runner/go/pkg/mod/github.com/pingcap/errors@v0.11.5-0.20211224045212-9687c2b0f87c/juju_adaptor.go:15\ngithub.com/pingcap/tidb/br/pkg/task.waitUntilAllScheduleStopped.func1\n\t/home/runner/work/ebs-snap-images/ebs-snap-images/br/pkg/task/backup_ebs.go:391\ngithub.com/pingcap/tidb/br/pkg/utils.(*WorkerPool).ApplyOnErrorGroup.func1\n\t/home/runner/work/ebs-snap-images/ebs-snap-images/br/pkg/utils/worker.go:76\ngolang.org/x/sync/errgroup.(*Group).Go.func1\n\t/home/runner/go/pkg/mod/golang.org/x/sync@v0.0.0-20220722155255-886fb9371eb4/errgroup/errgroup.go:75\nruntime.goexit\n\t/opt/hostedtoolcache/go/1.18.5/x64/src/runtime/asm_amd64.s:1571"]